Server/src/lib/chat/moderator.ts

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
this is chat only - not sure if we want to remove all the chat routes ?

Reply:
Created as work item
https://skype.visualstudio.com/SPOOL/_workitems/edit/2693385